page.go

```diff
@@ -164,6 +166,16 @@ func (p *Page) ViewportE(params *proto.EmulationSetDeviceMetricsOverride) error
 	return err
 }
 
+// GetViewport sets and returns the current viewport
+func (p *Page) GetViewport() (*proto.PageVisualViewport, error) {
```

Comment:
Please read the [E suffix](https://github.com/ysmood/rod/blob/master/.github/CONTRIBUTING.md#e-suffix) section and change the name accordingly.